zetta_utils/mazepa_layer_processing/common/subchunkable_apply_flow.py

```diff
@@ -214,12 +219,33 @@ def build_subchunkable_apply_flow(  # pylint: disable=keyword-arg-before-vararg,
             )
         bbox_ = bbox
 
+    defer_blend = processing_blend_modes == "defer" or processing_blend_modes[-1] == "defer"
+
     if dst is None:
+        if defer_blend:
+            raise ValueError('`dst` cannot be None when `processing_blend_modes` is "defer".')
         if not skip_intermediaries:
-            raise ValueError("`skip_intermediaries` must be True when `dst` is None.")
+            raise ValueError(
+                "`skip_intermediaries` must be True when `dst` is None and "
+                "`processing_blend_modes` is not "
+                "defer"
+                "."
```

Comment:
Combine lines? Linters do this sometimes....

Reply:
Yeah I definitely didn't do that!  Probably `black`'s fault.  Looks like it took out the intended quotation marks, too.  I'll see what I can do.